packages/node/package.json

```diff
@@ -24,10 +24,10 @@
     "@nestjs/event-emitter": "^2.0.0",
     "@nestjs/platform-express": "^9.4.0",
     "@nestjs/schedule": "^3.0.1",
-    "@subql/common": "dev",
+    "@subql/common": "3.1.2",
     "@subql/common-concordium": "workspace:*",
-    "@subql/node-core": "5.0.2-1",
-    "@subql/testing": "^2.0.0",
+    "@subql/node-core": "5.0.4-2",
+    "@subql/testing": "2.0.3-0",
```

Comment:
Why are these pre-release versions?

Reply:
updated node-core to latest release, it looks like it's using the dev release on @subql/testing. so here we are using the same.